src/main/resources/application.yaml

```diff
@@ -2,6 +2,9 @@
 # please keep properties in the alphabetical order
 
 stargate:
+  database:
+    limits:
+      maxCollections: 5
```

Comment:
Not necessarily strictly speaking (Config class has defaults) but added as documentation.